src/crypto/public_key/rsa.js

```diff
@@ -264,9 +273,6 @@ async function bnSign(hashAlgo, n, d, hashed) {
   n = uint8ArrayToBigInt(n);
   const m = uint8ArrayToBigInt(emsaEncode(hashAlgo, hashed, byteLength(n)));
   d = uint8ArrayToBigInt(d);
-  if (m >= n) {
-    throw new Error('Message size cannot exceed modulus size');
-  }
```

Comment:
The comment above makes me think we should still keep this error, for edge cases where the digest size fits but the encoded message size doesn't?

Reply:
`emsaEncode` already checks the length and throws if needed, this is an unreachable case

Reply:
OK :+1: